include/mxnet/kvstore.h

```diff
@@ -388,6 +397,22 @@ class KVStore {
   std::string type_;
 
   /**
+   * \brief Specifies whether or not to use compressed gradients
+   * Can be `none` or `2bit` for now
+   */
+  std::string compress_ = "none";
+
+  /**
+   * \brief positive threshold for 2bit compression
+   */
+  float pos_threshold_ = 0.1;
```

Comment:
Shouldn't have a const here

Reply:
Are you saying I shouldn't initialize it with a const here? Should I only initialize it through the constructor ( or its default arguments) ?

Reply:
you can set it to 0 here.
this 0.1 magic number that's never used is confusing.